core/src/modules/ops/exportProcessor.ts

```diff
@@ -0,0 +1,145 @@
+import Moment from "moment";
+import { Export } from "../../models/Export";
+import { Destination } from "../../models/Destination";
+import { CLS } from "../cls";
+import { Op } from "sequelize";
+import { ExportProcessor } from "../../models/ExportProcessor";
+import { config } from "actionhero";
+import { plugin } from "../plugin";
+
+export namespace ExportProcessorOps {
+  const defaultExportProcessingDelay = 1000 * 60 * 5;
+
+  export async function getExportsToProcess(exportProcessor: ExportProcessor) {
+    return Export.findAll({
+      where: { exportProcessorId: exportProcessor.id, state: "processing" },
+    });
+  }
```

Comment:
👍 You probably will be OK without a LIMIT here, as the batches we used to crate the exports are already limited by a setting (default: 100)